src/coreclr/jit/emitxarch.cpp

```diff
@@ -9918,6 +9921,11 @@ void emitter::emitDispIns(
             {
                 printf("L_M%03u_" FMT_BB, emitComp->compMethodID, id->idAddr()->iiaBBlabel->bbNum);
             }
+
+            if (((instrDescJmp*)id)->idjIsJmpAlways)
+            {
+                printf(" ; BB_ALWAYS");
+            }
```

Comment:
Do you want to keep this?

Reply:
Can you paste a sample output from it? Is it for disasm or dump?

Reply:
I added this to make the dump in https://github.com/dotnet/runtime/pull/69041#discussion_r878380386 show that there were two always jumps at the end. It's only visible in jit dump/disasm.

Reply:
Ok, keep it then.